code/game/objects/structures/morgue.dm

```diff
@@ -10,10 +10,14 @@
 
 	var/obj/structure/morgue_tray/connected_tray
 
+/obj/structure/morgue/get_mechanics_info()
+	return "[..()]<BR>Can be labeled once with a hand labeler."
```

Comment:
Should this be down somewhere lower behind a `has_extension(src, /datum/extension/labels/single)`?

Reply:
Tbh, I probably don't need to mention it really. Its just because it used to be done with a pen. And I wanted to communicate that to people trying to do it with a pen again and wondering what was going on.
If we move that to a base class, that would need to be in its own PR though.